Sources/Basics/ProgressAnimation/NinjaProgressAnimation.swift

```diff
@@ -0,0 +1,101 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
```

Comment:
nit: in the original file where this was copied from the copyright years are `2014 - 2018`
```suggestion
// Copyright (c) 2014-2024 Apple Inc. and the Swift project authors
```